src/Symfony/Component/Lock/Tests/Store/RetryTillSaveStoreTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Tests\Store;
+
+use Symfony\Component\Lock\Store\RedisStore;
+use Symfony\Component\Lock\Store\RetryTillSaveStore;
+
+/**
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class RetryTillSaveStoreTest extends AbstractStoreTest
+{
+    use BlockingStoreTestTrait;
+
+    public function getStore()
+    {
+        $redis = new \Predis\Client('tcp://'.getenv('REDIS_HOST').':6379');
```

Comment:
This part of the test is mostly a copy/paste of the RedisAdapterTest from the Cache component.
The variable is define is the `phpunit.xml` file and should exists.

I don't think it worst it to check whether or not someone destroy the config file and expect a successful result.